python/pdstools/__init__.py

```diff
@@ -10,22 +10,53 @@
 from pathlib import Path
 
 from .adm.ADMDatamart import ADMDatamart
-from .adm.ADMTrees import ADMTrees, MultiTrees
-from .adm.BinAggregator import BinAggregator
-from .decision_analyzer import DecisionData
-from .pega_io import API, S3, Anonymization, File, get_token, readDSExport
-from .pega_io.API import setupAzureOpenAI
-from .prediction import Prediction
-from .utils import NBAD, cdh_utils, datasets, errors
-from .utils.cdh_utils import defaultPredictorCategorization
-from .utils.CDHLimits import CDHLimits
-from .utils.datasets import CDHSample, SampleTrees, SampleValueFinder
-from .utils.polars_ext import *
-from .utils.show_versions import show_versions
-from .utils.table_definitions import PegaDefaultTables
-from .valuefinder.ValueFinder import ValueFinder
+
+# from .utils.polars_ext import *
+# from .utils.show_versions import show_versions
+# from .valuefinder.ValueFinder import ValueFinder
+from .infinity import Infinity
```

Comment:
I need httpx when importing pdstools because it imports Infinity which requires httpx. @StijnKas

Reply:
Good spot @yusufuyanik1 . Need to make sure https stays optional :)